### DISCUSSION TITLE
Changing BPM or Offset of the red Timing Point when paused shouldn't move the user's timeline back where this Timing Point is

### DISCUSSION THREAD

**Discussion by Skubers** [ppy/osu#18741] [Ideas] [answered]
If I'd like to adjust BPM and Offset in a specific section of a song (because it would have much more audible music themes there), itshouldn't scroll me back to the Timing Point. It's already a thing in Stable and I think it is such a no-brainer for many mappers that it's a real inconvenience in Lazer

To reproduce, just go into a map in editor and put a Red Timing Point, then scroll somewhere further and pause the song. Try to adjust the Offset or BPM and see that it scrolls you back to the timing point. This isn't a case in Stable

**Comment by peppy** [comment]
Can you explain the use case for not wanting to see where... the offset is actually at? I'd like to better understand your workflow, because this sounds counter-intuitive to me at first glance.

Are you using the new waveform display? If so, do you know that you can lock it to a specific time by clicking on it?

**Reply by Skubers** [reply]
Give me a sec, I'll provide an example

**Reply by Skubers** [reply]
https://user-images.githubusercontent.com/82095491/174197474-240d455a-86d5-4373-b62d-5d10f4a8c97f.mp4

Here's an example of me trying to use a later part of the song to distinguish offset. I know I could just play the song and adjust when it plays, but yeah, if you want to stop and think for a second and you change the offset, you go back to the start.
Also for some reason my waveforms are not appearing (?)

**Reply by Theighlin** [reply]
Do the waveforms never appear or does relaunching the game fix it temporarily? Because if this always happens, you're welcome to upload your logs at #18479

**Reply by apollo-dw** [reply]
This behaviour does seem annoying, often times the song offers more stronger, discernable beats later in the song which are easier to gauge offset against.

**Reply by peppy** [reply]
Thanks for the example. I agree that looks broken.

**Comment by peppy** [comment, marked as answer]
Will be fixed via https://github.com/ppy/osu/pull/18762.
